loader/language.cpp

```diff
@@ -95,14 +95,14 @@ const MultiLang lang_fav_added[] = {
 	{LANG_EN, "Added to favorites"},
 	{LANG_DE, "Favorit hinzugef�gt"},
 	{LANG_FR, "Ajout� aux favoris"},
-	{LANG_ES, "Favorito agregado"},
+	{LANG_ES, "Agregado a favoritos"},
```

Comment:
This text is one character too long, which is why, in the german translation e.g. i wronge "Favorite added" since that was shorter